lib/TrackInfo.js

```diff
@@ -238,38 +238,53 @@ class TrackInfo
 
 /**
  * Expands a plain JSON object containing an TrackInfo
- * @param {TrackInfoPlain} plain JSON object
+ * @param {TrackInfoPlain | TrackInfo} plain JSON object
  * @returns {TrackInfo} Parsed Track info
  */
 TrackInfo.expand = function(plain)
 {
+	//Ensure it is not already a TrackInfo object
+	if (plain.constructor.name === "TrackInfo")
+		// @ts-ignore
+		return plain;
+
 	//Create new
 	const trackInfo =  new TrackInfo(plain.media,plain.id);
 	//Check mediaId
 	if (plain.mediaId)
 		//Set it
 		trackInfo.setMediaId(plain.mediaId);
 	//Gor each ssrc
-	for (let i=0;plain.ssrcs && i<plain.ssrcs.length;++i)
+	for (const ssrc of plain.ssrcs || [])
```

Comment:
question, is `plain.ssrcs` allowed to not be present?

Reply:
on previous code, it was

Reply:
the field was marked mandatory in the type. correcting to optional then.